ReactNativeTemplate/ios/Podfile

```diff
@@ -1,5 +1,4 @@
-# Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '10.0'
```

Comment:
I will do that - devs could change the podfile and rerun pod update it they want 9.0.